### ISSUE TITLE
Winget lists software with 'source=winget' even though it was not installed with winget

### ISSUE THREAD

**Issue by ShrykeWindgrace** [microsoft/winget-cli#2546]
### Brief description of your issue

Apps installed by other package managers or by their native installers are listed by winget with `source=winget`.

### Steps to reproduce

- install vim with chocolatey `choco install vim -y`
- `winget list -s winget | grep vim` has a hit

### Expected behavior

Winget reports such apps as having no source or specifying the original installation method.

### Actual behavior

`winget` thinks that it has installed `vim`, even though it did not.

### Environment

```shell
winget --info
Windows Package Manager v1.3.2091
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.2006
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2091.0
```

**Comment by Trenly** [comment]
The source selector isn't stating which source it was installed from. Rather, it is saying that vim is available through the winget source

**Comment by ShrykeWindgrace** [comment]
@Trenly I see, thank you for clarification. In this case, what would the correct way to query for packages _installed_ by winget (and eventually MSStore)?

**Comment by Trenly** [comment]
> @Trenly I see, thank you for clarification. In this case, what would the correct way to query for packages _installed_ by winget (and eventually MSStore)?

I don't believe that functionality exists yet, as Winget looks at all the applications installed on a machine to try and map them for updates
